### PR TITLE
v1.28 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.28.0 Release
+- Release Date: February 16, 2020
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
```
Comment (enahum) [inline CHANGELOG.md]: update ESR version

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
+### Highlights
+
+#### UI/UX Improvements to the Post Draft Area
+
+#### Deep Linking
+ - Links to posts in email notifications now launch to a browser landing page with option to open in the Desktop or Mobile app.
```
Comment (enahum) [inline CHANGELOG.md]:
```suggestion
 - Links to posts in email notifications now launch to a browser landing page with option to open in the Mobile app.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+
+### Improvements
+ - Added a post menu item to mark posts as unread.
```
Comment (enahum) [inline CHANGELOG.md]: I think this is not part of this version but an older one, or am I wrong?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+ - Removed markdown rendering from Channel Purpose in channel info screen.
+ - Improved channel info transition so that it opens up as a modal rather than as a drawer from the right.
+ - Clicking on the time in the iOS status bar now scrolls up the center channel.
+ - Improved the left-hand sidebar on iOS so that it slides into place as the sidebar is opened while becoming visible as content over it is moved.
```
Comment (enahum) [inline CHANGELOG.md]: can we rephrase this?

File: CHANGELOG.md (hunk 3 above)
Comment (amyblais) [reply CHANGELOG.md]: The ticket and PR are marked for v1.28:
 - https://github.com/mattermost/mattermost-mobile/pull/3202
 - https://mattermost.atlassian.net/browse/MM-17068

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,38 @@
...
+- Server Versions Supported: Server v5.19+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
+### Highlights
+
+#### UI/UX Improvements to the Post Draft Area
+
```
Comment (adamjclarkson) [inline CHANGELOG.md]: Can we add a description of the changes? Let me know if you need a couple sentences.

Comment (amyblais) [reply CHANGELOG.md]: @adamjclarkson Yes please, if you can help with a couple sentences that would be great.

File: CHANGELOG.md (hunk 3 above)
Comment (esethna) [reply CHANGELOG.md]: They are definitely not for this release. This was v1.26

**Comment by amyblais** [comment]
@enahum Can you help merge this?

**Pull request merged**